studio/schemas/documents/locale.ts

```diff
@@ -0,0 +1,41 @@
+import { defineField, defineType } from "sanity";
+
+import { relevantCurrencies } from "studio/utils/currencies";
+import { relevantLocales } from "studio/utils/locales";
+
+export const localeID = "locale";
+
+const locale = defineType({
+  name: localeID,
+  type: "document",
+  fields: [
+    defineField({
+      name: "locale",
+      type: "string",
+      title: "Locale",
+      description: "Select the most relevant locale for the website.",
```

Comment:
Not sure "locale" means something to an editor

Reply:
Changed to "Region" now. Possible alternatives could be "Regional Settings", "Format Region", "Region Formats", "Number Formatting". Any other suggestions?